swift/Sources/Keystore/KeyStore.swift

```diff
@@ -11,12 +11,25 @@ public final class KeyStore {
     /// The key file directory.
     public let keyDirectory: URL
 
+    /// The watches file URL.
+    public let watchesFile: URL?
+
     /// List of wallets.
     public private(set) var wallets = [Wallet]()
 
+    /// List of accounts being watched
+    public var watches = [Watch]()
+
     /// Creates a `KeyStore` for the given directory.
-    public init(keyDirectory: URL) throws {
+    public init(keyDirectory: URL, watchesFile: URL? = nil) throws {
```

Comment:
we can also re use `keyDirectory: URL` directory and store watch files in `watch.json` to make it simpler

Reply:
Done